packages/@react-aria/tag/test/useTagGroup.test.js

```diff
@@ -96,17 +96,15 @@ describe('useTagGroup', function () {
     expect(tags[3]).toHaveAttribute('aria-selected', 'true');
     expect(tags[4]).toHaveAttribute('aria-selected', 'false');
 
-    fireEvent.keyDown(tags[3], {key: 'Backspace'});
-    fireEvent.keyUp(tags[3], {key: 'Backspace'});
-
+    await user.keyboard('{Backspace}');
     expect(onRemove).toHaveBeenCalledTimes(1);
     expect(onRemove).toHaveBeenLastCalledWith(new Set(['parking', 'pool']));
 
-    fireEvent.keyDown(tags[0], {key: 'Backspace'});
-    fireEvent.keyUp(tags[0], {key: 'Backspace'});
+    await user.click(tags[0]);
+    await user.keyboard('{Backspace}');
 
     expect(onRemove).toHaveBeenCalledTimes(2);
-    expect(onRemove).toHaveBeenLastCalledWith(new Set(['laundry']));
+    expect(onRemove).toHaveBeenLastCalledWith(new Set(['laundry', 'parking', 'pool']));
```

Comment:
how did this pass earlier? i'm guessing it's called with all three because we don't actually remove the previous selected tags?

Reply:
you know, I don't actually recall how this used to pass because as you've noted the test component doesn't actually remove any of the tags with its `onRemove`. Lemme poke around and see if I can find out

Reply:
Ah, I remember now. So the old test was firing keydown directly on the "laundry" tag which doesn't actually cause it to be selected, hence why the backspace operation only triggers for the single item via https://github.com/adobe/react-spectrum/blob/46ae3314a2e2c1ab856c82c8d9dc941e8c70016e/packages/%40react-aria/tag/src/useTag.ts#L69-L70. I simply modified the test to fall more inline with the "should support selection" intent of this test case so that the "laundry" tag is actually selected before `Backspace` is pressed.